draft-bbs-signatures.md

```diff
@@ -730,14 +733,20 @@ Procedure:
 5. return result
 ```
 
-### Hash to scalar
+### Hashing to scalars
+
+This operation describes how to hash an arbitrary octet string to `n` scalar values in the multiplicative group of integers mod r (i.e., values in the range [1, r-1]).  This procedure acts as a helper function, and it is used internally in various places within the operations described in the spec. To map a message to a scalar that would be passed as input to the [Sign](#sign), [Verify](#verify), [ProofGen](#proofgen) and [ProofVerify](#proofgen) functions, one must use [MapMessageToScalarAsHash](#mapmessagetoscalar) instead.
+
+This document defines two different hash_to_scalar operations
```

Comment:
hash_to_scalar needs escaping, or maybe say something like "two different operations for hashing inputs into scalar values"

Reply:
Updated! Thank you!